App/features/group/reducer.ts

```diff
@@ -17,9 +19,19 @@ export interface GroupState {
 
 export type GroupAction = ActionType<typeof actions>
 
-export default combineReducers<GroupState>({
+const persistConfig: PersistConfig = {
```

Comment:
Using a nested persist config to control what is being persisted in a nested reducer where some things need to be persisted but not others.